Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Person.h"
+
+#include <iostream>
+
+Person::Person()
+{
+  this -> name_ = "";
```

Comment:
This goes for all pointers in this PR, but I think `this->` isn't really necessary as it's generally implied.

We generally use the `_` at the end of class data members (see Google's [Variable Names guide](https://google.github.io/styleguide/cppguide.html#Variable_Names)) to signify that it's part of the class, as it's a lot shorter then `this->` and generally makes things easier.